Inventory/Controllers/APIController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using InventoryLogic.Facade;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InventoryAPI.Controllers
+{
+    [Route("api/v{v:apiVersion}/[controller]")]
+    [ApiController]
+    [ApiVersion("0.1")]
+    public class APIController<Type> : ControllerBase
+    {
+        private readonly IFacade<Type> facade;
+
+        public APIController(IFacade<Type> facade)
+        {
+            this.facade = facade;
+        }
+
+        /// <summary>
+        /// List of <typeparamref name="Type"/> definitions
+        /// </summary>
```

Comment:
Lijkt erop dat de type niet in swagger zichtbaar wordt bij mij, dus dit lijkt niet te werken?